packages/react/src/useForm.ts

```diff
@@ -18,7 +18,7 @@ export interface InertiaFormProps<TForm extends FormDataType> {
   wasSuccessful: boolean
   recentlySuccessful: boolean
   setData: setDataByObject<TForm> & setDataByMethod<TForm> & setDataByKeyValuePair<TForm>
-  transform: (callback: (data: TForm) => TForm) => void
+  transform: (callback: (data: TForm) => object) => void
```

Comment:
The return value cannot be of type `TForm` as that defeats the purpose of transform. It should be anything but TForm. 😄